src/services/infra/InfraService.ts

```diff
@@ -234,15 +228,14 @@ export default class InfraService {
         domainValidationTarget,
         requestorEmail: requestor.email ? requestor.email : "",
         agencyEmail: agency.email ? agency.email : "", // TODO: remove conditional after making email not optional/nullable
-        success: true,
       }
 
-      if (newLaunchParams.redirectionDomainSource) {
+      if (redirectionDomain) {
         message.redirectionDomain = [
           {
-            source: newLaunchParams.redirectionDomainSource,
-            target: primaryDomainTarget,
-            type: this.isRootDomain(primaryDomain) ? "CNAME" : "A",
+            source: redirectionDomain,
+            target: REDIRECTION_SERVER_IP,
+            type: "A",
```

Comment:
Arh. Previously, my assumption was that we will be using a direct CNAME from 
blah.moe.edu.sg -> blah.cloudfront.net.
However, since we want to allow for the forward compatibility of schools using their emails for their root domains , we no longer do this flow. Instead we use the A records for this! (similar to our typical agency flow)

Reply:
@seaerchin hey, just checking in, is this clear?